### DISCUSSION TITLE
Use `let` for class field declarations

### DISCUSSION THREAD

**Discussion by eladb** [winglang/wing#5162] [Ideas]
[This](https://github.com/winglang/wing/issues/5150) and a few conversations we've had around a more "functional" class declarations made me think that maybe we should consider using `let` for field definitions:

```js
class Foo {
  let x: num;
  pub let y: str;
}
```

instead of:

```js
class Foo {
  x: num;
  pub y: str;
}
```

Maybe this can even be used for methods:

```js
class Foo {
  let myField: num;

  new() {
    
  }

  pub let myMethod = (x: str) => {
    return "this is a method";
  };
}
```

Thoughts?

**Comment by staycoolcall911** [comment]
The `let` feels redundant, but maybe I'm just used to how it is today. Can you elaborate on why you're considering adding it? Aesthetic reasons?
